cytosnake/cli/cmd.py

```diff
@@ -6,17 +6,19 @@
 Generates CLI interface in order to interact with CytoSnake.
 """
 import logging
+import pathlib
 import sys
-from pathlib import Path
 
-# cytosnake imports
 from cytosnake.cli.args import CliControlPanel
 from cytosnake.cli.cli_docs import cli_docs, init_doc, run_doc
 from cytosnake.cli.exec.workflow_exec import workflow_executor
 from cytosnake.cli.setup_init import init_cp_data, init_dp_data
 from cytosnake.common.errors import ProjectExistsError, WorkflowFailedException
 from cytosnake.guards.input_guards import check_init_parameter_inputs
 from cytosnake.utils import cyto_paths
+
+# cytosnake imports
```

Comment:
Consider removing this comment or placing it back where it was located before these changes.